platform-includes/performance/configure-sample-rate/python.mdx

```diff
@@ -1,4 +1,10 @@
-Activate performance monitoring by setting `enable_tracing` to `True`. Performance Monitoring is available for the Sentry Python SDK version ≥ 0.11.2.
+Activate performance monitoring by setting `enable_tracing` to `True`.
+
+<Note>
+
+`enable_tracing` is available in Sentry Python SDK version ≥ 1.16.0. To enable tracing in older SDK versions (≥ 0.11.2), use `traces_sample_rate=1.0`.
+
+</Note>
```

Comment:
```suggestion



Note that `enable_tracing` is available in Sentry Python SDK **version ≥ 1.16.0**. To enable tracing in older SDK versions (≥ 0.11.2), use `traces_sample_rate=1.0`.


```
There are so many notes in this section of docs that it's more impactful to just make this part of the regular text :)

Reply:
Makes sense :)